bjjdestroyer/overlook-hotel
# Refactor Code

#### What’s this PR do?
- Fix issues found with lint
- Reorganize css to be clearer
- Add responsiveness to booking page
- General css refactoring
#### Where should the reviewer start?
- Look through all CSS files
#### How should this be manually tested?
- in terminal, run npm start
- go to local host, sign in as either manager or customer
- as customer, go to booking page and play with the size of the screen to see responsivness in action
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
